jack-and-the-beans/buddies-ios
# BUD-134 show view activity when joined

This adds the ability to see the activity details while you are a member.

- Animates up when you join
- Animates down when you tap the mini description at the top
- Animates up when you tap the little up button
- See date, location, and picture in the mini description area.

The architecture should be set up such that the next functionality (e.g. delete user) should be easy to add.

_Note: all of the view activity code is in its own subdirectory now per @lukemeierj's request_

To test:
1. pull code and run
2. Go to topics and select an activity
    - If you are a member, the chat view should indicate that
    - If you are an owner, the chat view should indicate that
    - You should see a mini area if you have joined, and tap it to reveal the description (with a "close" button instead of a join button)
    -  If you are NOT a member of the activity, it should still show the maximized description and allow you to join it. On joining, there should be an animation into the joined activity view.
